docs/sources/howtos/bindings.md

```diff
@@ -60,11 +60,12 @@ Eclipse SmartHome allows you to define your *ThingTypes* in a declarative way th
         <channel id="temperature" typeId="temperature" />
     </channels>
     <config-description>
-        <parameter name="location" type="text" required="true">
```

Comment:
We changed "required" to be an attribute. The xml element remained just for backward compatibility. But some bindings needs to be adapted.